nginx.master.conf

```diff
@@ -285,35 +285,42 @@ http {
             proxy_pass http://log/;
         }
 
-        location ~ ^/system/v1/agent/(?<agentid>[0-9a-zA-Z-]+)/logs/v1/(?<url>.*)$ {
+        location /system/v1/metrics/ {
+            access_by_lua 'auth.validate_jwt_or_exit()';
+
+            include common/proxy-headers.conf;
+            proxy_pass http://metrics/;
+        }
+
+		# Ordering is important for these two endpoints!!!
+        location ~ ^/system/v1/agent/(?<agentid>[0-9a-zA-Z-]+)(?<type>(/metrics/v0|/logs/v1))(?<url>.*)$ {
```

Comment:
Could you please double-check ? A fragement from @philipnrmn  PR:
```
         location ~ ^/system/v1/agent/(?<agentid>[0-9a-zA-Z-]+)/logs/v1/(?<url>.*)$ {
             access_by_lua 'auth.validate_jwt_or_exit()';
             set $agentaddr '';
             more_clear_input_headers Accept-Encoding;
             rewrite ^/(slave|agent)/[0-9a-zA-Z-]+/.*$ $url break;
             rewrite_by_lua_file conf/master/agent.lua;
 
             include common/server-sent-events.conf;
             include common/proxy-headers.conf;
 
             proxy_pass http://$agentaddr:61001/system/v1/logs/v1/$url$is_args$query_string;
         }
 
(...)

        location ~ ^/system/v1/agent/(?<agentid>[0-9a-zA-Z-]+)/metrics/v0/(?<url>.*)$ {
            access_by_lua 'auth.validate_jwt_or_exit()';
            set $agentaddr '';
            more_clear_input_headers Accept-Encoding;
            rewrite ^/(slave|agent)/[0-9a-zA-Z-]+/.*$ $url break;
            rewrite_by_lua_file conf/master/agent.lua;

            include common/server-sent-events.conf;
            include common/proxy-headers.conf;

            proxy_pass http://$agentaddr:61001/system/v1/metrics/v0/$url$is_args$query_string;
        }
```

I have concatenated only `/logs/v1` and `/metrics/v0` - they seem identical. Please correct me if I am wrong.